jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -683,24 +922,54 @@ Function Write-ToLog {
         # Format Date for our Log File
         $FormattedDate = Get-Date -Format "yyyy-MM-dd HH:mm:ss"
         # Write message to error, warning, or verbose pipeline and specify $LevelText
-        Switch ($Level) {
-            'Error' {
-                Write-Error $Message
-                $LevelText = 'ERROR:'
-            }
-            'Warn' {
-                Write-Warning $Message
-                $LevelText = 'WARNING:'
+        if ($Script:AdminDebug) {
+            Switch ($Level) {
+                'Error' {
+                    Write-Error $Message
+                    $LevelText = 'ERROR:'
+                }
+                'Warn' {
+                    Write-Warning $Message
+                    $LevelText = 'WARNING:'
+                }
+                'Info' {
+                    Write-Verbose $Message
+                    $LevelText = 'INFO:'
+                }
+                'Verbose' {
+                    Write-Verbose $Message
+                    $LevelText = 'INFO:'
+                }
             }
-            'Info' {
-                Write-Verbose $Message
-                $LevelText = 'INFO:'
+        } else {
+            Switch ($Level) {
+                'Error' {
+                    Write-Error $Message
+                    $LevelText = 'ERROR:'
+                }
+                'Warn' {
+                    $LevelText = 'WARNING:'
+                }
+                'Info' {
+                    $LevelText = 'INFO:'
+                }
+                'Verbose' {
+                    Write-Verbose $Message
+                    $LevelText = 'INFO:'
+                }
             }
         }
+
+        # Add the message to the log messages and space down
+        $logMessage = "$FormattedDate $LevelText $Message" + "`r`n"
+        if ($Script:ProgressBar) {
+            Update-LogTextBlock -LogText $logMessage -ProgressBar $Script:ProgressBar
+        }
         # Write log entry to $Path
         "$FormattedDate $LevelText $Message" | Out-File -FilePath $Path -Append
```

Comment:
I think we need to encode the message content a bit differently. The log file displays correctly within the GUI but there's a space between each character in the actual log. 

![Screenshot 2024-07-12 at 2 08 20 PM](https://github.com/user-attachments/assets/c8146102-aef3-421d-bd7f-c6e31c25e56d)

Reply:
I'm not seeing this on my end. Maybe @gweinjc can chime in?